exec/jdbc-all/pom.xml

```diff
@@ -548,7 +548,7 @@
                   This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
 
                   </message>
-                  <maxsize>46600000</maxsize>
+                  <maxsize>47200000</maxsize>
```

Comment:
Are you sure that these newly added libraries should be included in the JDBC jar? I think it is better to exclude them from here...